### ISSUE TITLE
Error: Call to undefined method helper_plugin_diffpreview_changes::getInfo()

### ISSUE THREAD

**Issue by glensc** [tqdv/dokuwiki-diffpreview#5]
```
Error: Call to undefined method helper_plugin_diffpreview_changes::getInfo()
An unforeseen error has occured. This is most likely a bug somewhere. It might be a problem in the info plugin.

More info has been written to the DokuWiki error log.
```

I was editing a wiki page, and suddenly I get this error, and it doesn't go away after page reloads.

Using: v1.3.0 # 2022-11-24

Update: this is caused by enabling `~~INFO:helperplugins~~` on wiki page.

**Comment by glensc** [comment]
Trace from error log

```
2023-06-01 11:06:08     /usr/share/dokuwiki/lib/plugins/info/syntax.php(136)    Error: Call to undefined method helper_plugin_diffpreview_changes::getInfo()
  #0 /usr/share/dokuwiki/lib/plugins/info/syntax.php(93): syntax_plugin_info->renderPlugins('helper', Object(Doku_Renderer_xhtml))
  #1 /usr/share/dokuwiki/inc/parser/renderer.php(119): syntax_plugin_info->render('xhtml', Object(Doku_Renderer_xhtml), Array)
  #2 /usr/share/dokuwiki/inc/parserutils.php(700): Doku_Renderer->plugin('info', Array, 5, '~~INFO:helperpl...')
  #3 /usr/share/dokuwiki/inc/parserutils.php(158): p_render('xhtml', Array, NULL)
  #4 /usr/share/dokuwiki/inc/parserutils.php(96): p_cached_output('/var/lib/dokuwi...', 'xhtml', 'service:dokuwik...')
  #5 /usr/share/dokuwiki/inc/Ui/PageView.php(68): p_wiki_xhtml('service:dokuwik...', 0, true, '')
  #6 /usr/share/dokuwiki/inc/Action/Show.php(37): dokuwiki\Ui\PageView->show()
  #7 /usr/share/dokuwiki/inc/template.php(100): dokuwiki\Action\Show->tplContent()
  #8 [internal function]: tpl_content_core('show')
  #9 /usr/share/dokuwiki/inc/Extension/Event.php(133): call_user_func_array('tpl_content_cor...', Array)
  #10 /usr/share/dokuwiki/inc/Extension/Event.php(199): dokuwiki\Extension\Event->trigger('tpl_content_cor...', true)
  #11 /usr/share/dokuwiki/inc/template.php(85): dokuwiki\Extension\Event::createAndTrigger('TPL_ACT_RENDER', 'show', 'tpl_content_cor...')
  #12 /usr/share/dokuwiki/lib/tpl/dokuwiki/main.php(59): tpl_content()
  #13 /usr/share/dokuwiki/inc/actions.php(27): include('/usr/share/doku...')
  #14 /usr/share/dokuwiki/doku.php(126): act_dispatch()
  #15 {main}
(END)
```

**Comment by glensc** [comment]
ok. i made typo in edit that made the comment plugin `/*` mark not be reached:

```
                                                                             
/* disabled
The following **helper** plugins are available in this particular DokuWiki install
~~INFO:helperplugins~~
*/  
```

so, it started to render `~~INFO:helperplugins~~` and that will fail here.

**Comment by tahyphen** [comment]
The same error occurred to me after upgrading DokuWiki.

My environment is as below.

- PHP 7.3.31-1~deb10u3
- DokuWiki 2023-04-04a (upgraded from 2022-07-31a)
- diffpreview 2022-11-24 (v1.3.0)

Compared with other helper plugins, I found a possible fix.
This fix is by editing [`helper_plugin_diffpreview_changes` class](https://github.com/tqdv/dokuwiki-diffpreview/blob/master/helper/changes.php#L7) as the following code.

```php
class helper_plugin_diffpreview_changes extends Dokuwiki_Plugin {
```

Although I am unfamiliar with plugin development / PHP, this patch could be a help.

**Comment by tqdv** [comment]
Thanks for the fix @tahyphen.
I updated the plugin and released version 2023-06-18 (v1.3.1).
